src/main/java/org/jsmart/zerocode/core/engine/assertion/ArraySizeAsserter.java

```diff
@@ -27,7 +27,10 @@ public AssertionReport actualEqualsToExpected(Object result) {
                 return AssertionReport.createFieldMatchesReport();
             }
 
-            return AssertionReport.createFieldDoesNotMatchReport(path, String.format("Array of size %d", expectedSize), result);
+            return AssertionReport.createFieldDoesNotMatchReport(
```

Comment:
@nirmalchandra , this should go as a separate PR, or as a separate commit.
Do not mix up the commits for different issues. Please!